code/modules/projectiles/gun.dm

```diff
@@ -117,7 +117,10 @@
 	if(!pinless)
 		if(pin)
 			. += "It has \a [pin] installed."
-			. += span_info("[pin] looks like it could be removed with some <b>tools</b>.")
+			if(pin.pin_removable)
+				. += span_info("[pin] looks like it could be removed with some <b>tools</b>.")
+			else
+				. += span_info("[pin] looks like its firmly locked in, it looks impossible to remove.")
```

Comment:
```suggestion
			if(pin.pin_removable)
				. += span_info("[pin] looks like [pin.p_they()] could be removed with some <b>tools</b>.")
			else
				. += span_info("[pin] looks like [pin.p_theyre()] firmly locked in, [pin.p_they()] looks impossible to remove.")
```

Usually don't care that much for something very specific like this but "its" is wrong because it's a contraction here, so we might as well